openstack/networking/v2/extensions/vpnaas/endpointgroups/requests.go

```diff
@@ -0,0 +1,58 @@
+package endpointgroups
+
+import "github.com/gophercloud/gophercloud"
+
+type EndpointType string
+
+const (
+	TypeSubnet  EndpointType = "subnet"
+	TypeCidr    EndpointType = "cidr"
+	TypeVlan    EndpointType = "vlan"
+	TypeNetwork EndpointType = "network"
+	TypeRouter  EndpointType = "router"
+)
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToGroupCreateMap() (map[string]interface{}, error)
```

Comment:
Since "group" is a rather ambiguous word, let's call this `ToEndpointGroupCreateMap`